Project.toml

```diff
@@ -16,6 +16,7 @@ Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
 REPL = "3fa0cd96-eef1-5676-8a61-b3b8758bbffb"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 Unicode = "4ec0a83e-493e-50e2-b9ac-8f72acf5a8f5"
+tectonic_jll = "d7dd28d6-a5e6-559c-9131-7eb760cdacc5"
```

Comment:
1. Can we make this dep optional? Not everyone wants to build a PDF.

Of course, if the optional dependencies are not easy to implement, adding a [~5MB][5MB] dependency is not unacceptable

2. Add `tectonic_jll` or [Tectonic.jl](https://github.com/MichaelHatherly/Tectonic.jl)?

[5MB]: https://github.com/JuliaBinaryWrappers/tectonic_jll.jl/releases/tag/tectonic-v0.8.0%2B0

Reply:
I was going to comment the same. I think you can do what I did for node in https://github.com/JuliaDocs/Documenter.jl/pull/1627

Reply:
What should the interface look like then? `LaTeX(platform="native", tectonic=tectonic_jll.tectonic())`? Or `LaTeX(platform="tectonic", tectonic=tectonic_jll.tectonic())`? Or something else?

Reply:
The latter, and passing `tectonic` can be ignored if you have it in `PATH` already.